cmd/snap-update-ns/main.go

```diff
@@ -107,10 +112,14 @@ func run() error {
 	// symlinks or perform other malicious activity (such as attempting to
 	// introduce a symlink that would cause us to mount something other
 	// than what we expected).
+	logger.Debugf("freezing processes of snap %q", snapName)
 	if err := freezeSnapProcesses(opts.Positionals.SnapName); err != nil {
 		return err
```

Comment:
Would it make sense to log this err too?

Reply:
The error gets printed to stderr when the program fails, this error is propagated directly out of `run` and into `main`.